adding_species_manuscript.md

```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
 # Examples from "Zoo" hackathon
 
     -   A species that was fully added (Bos taurus?) **GREGOR**
-
-        -   resources available
-        -   reasons for adding
-
-             - ag species, hence important for breeding and genetic simulations of applied problems, wild ??? extinct
-            
-            - Ne inferred to very recent times using a specific haplotype method (@MacLeod2013)
-            - decreasing Ne due to domestication and recent intense selection, this scenario is quite different to many other cases (stuff like GWAS is very limited due to long-range LD, but genomic prediction works really well withing populations, ... cite @Meuwissen2001, @MacLeod2014, https://royalsocietypublishing.org/doi/abs/10.1098/rspb.2016.0569)
-
-        -   can be used for methods development and inference
+    
+We have added Bos Taurus (cattle) species as an example of an agricultural
+species. Agricultural species have generally experienced strong reduction in
+effective population size over time due to selection caused by domestication and
+recently selective breeding. Both of these processes have occurred over a
+relatively short period (~10,000 years or less) and are increasingly intensified
+to improve food production (@Gaut2018, @MacLeod2013). In cattle, nowadays, high
+quality genome assemblies are available for several breeds (e.g., @Rosen2020,
+@Heaton2021, @Talenti2022) and since the introduction of genomic prediction
+(@Meuwissen2001) the use of use of genomics is booming (e.g., @Taylor2016,
+@VanRaden2020). This boom inspired numerous simulation studies to study optimal
+uses of genomic data in selective breeding (@Meuwissen2001, @MacLeod2014, and
+others). Cattle is interesting for population genetic studies due to its
+decreasing effective population size. In the cosmopolitan dairy breed Holstein,
+the effective population size was estimated to be as low as ~90 around 1980
+(@MacLeod2013) and has since decreased even further due to intense selective
+breeding (e.g., @VanRaden2020, @Makanjouloa2020). This decrease is interesting
+because it is challenging the estimation of demography (@MacLeod2013),
+genome-wide association and prediction (@MacLeod2014), and inference of selection
+(@Hartfield2022). With respect to the parameters chosen in the stdpopsim
```

Comment:
I think we can consolidate the introductory paragraph here: Suggest replacing lines 343-355 with "In cattle, high quality genome assemblies are now available for several breeds and population genomic analyses have become widely used to improve selective breeding and genomic prediction. Their small effective population size (~90 around 1980, and continuing to decline due to intense selective breeding) makes demographic and selection inference, as well as genome-wide association and prediction, challenging. For these reasons, it was useful to develop a standardized population genomic cattle model for stdpopsim."

Reply:
(with appropriate citations of course)